tests/PutC/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <verilated_vcd_c.h>
+
+#define VL_DEBUG
+#include "Vtop.h"
+#include "verilated.h"
+
+static vluint64_t main_time = 0;
+
+double
+sc_time_stamp()
+{
+  return main_time;
+}
+
+int main (int argc, char **argv) {
+  Verilated::commandArgs(argc, argv);
+  Vtop *top = new Vtop();
+
+  Verilated::traceEverOn(true);
+  VerilatedVcdC* tfp = new VerilatedVcdC;
+  top->trace(tfp, 99);
+  tfp->open("dump.vcd");
+
+  while (!Verilated::gotFinish() && (main_time < 100)) {
+    top->eval();
+    tfp->dump(main_time);
+
+    main_time += 1;
+
+    std::cout << "time: " << main_time
+      << " a: " << (top->a?1:0)
+      << std::endl;
+  }
+  top->final();
+  tfp->close();
+    delete top;
+
+  return 0;
```

Comment:
The indentation seems to be a little bit off here

Reply:
Most lines are copied from https://github.com/alainmarcel/uhdm-integration/blob/master/tests/onenet/main.cpp.
I see that only line 37 has wrong indentation and I will correct it. I didn't notice it.